### PR TITLE
webapi: Pull data from database, run in same binary as goundstation

### PR DESCRIPTION
**Pull request by aDotInTheVoid**
Still needs to create the DB

### CODE REVIEW

**Comment by aDotInTheVoid** [comment]
I've switched away from toml config, as it's a faff for local development, where you don't want to have to edit a file to change settings.

File: cmd/webapi/main.go [hunk 1]
```diff
@@ -10,7 +10,8 @@ import (
 func main() {
```
Comment (danielg0) [inline cmd/webapi/main.go]: What's the aim of keeping a main for webapi now's it's been merged into control?

And won't it crash on it's first request due to the db being nil?

Comment (aDotInTheVoid) [reply cmd/webapi/main.go]: Oh, yeah, this can be deleted (groundstation too)

**Review by danielg0 — APPROVED** [review]
I'm fine with this PR, but I think we need to decide on a configuration format and stick to it. We now have:
- An unused toml implementation in `cmd/groundstation/config`
- Postgres unit tests that only use environment variables and fallback to a hardcoded value
- control.go that uses go flags and an environment variable that fallsback to the postgres default server

**Comment by danielg0** [comment]
To make myself clear, this is a topic for a discussion later, not now

**Pull request merged**